### PR TITLE
Hardcore Hunger, Buoy, Spawn, Beds

### PR DESCRIPTION
**Pull request by Koward**
Known issue : you can't jump when you're stuck at the bottom of water due to health penalty. This could be solved by primetoxinz's Forge PR. ATM I don't see how to do better.

### CODE REVIEW

**Comment by MRebhan** [comment]
Config option?

**Comment by Koward** [comment]
I will work on it. Should be easy to add, just condition to disable the events in HungerEventHandler.

File: src/main/java/betterwithmods/util/item/ItemExt.java [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+		buoyancy.put(Items.COOKED_PORKCHOP, 1.0F);
+		buoyancy.put(Items.PAINTING, 1.0F);
+		buoyancy.put(Items.SIGN, 1.0F);
+		buoyancy.put(Items.SADDLE, 1.0F);
+		buoyancy.put(Items.SNOWBALL, 1.0F);
+		buoyancy.put(Items.BOAT, 1.0F);
+		buoyancy.put("leather", 1.0F);
+		buoyancy.put(Items.PAPER, 1.0F);
+		buoyancy.put(Items.BOOK, 1.0F);
+		buoyancy.put("slimeBall", 1.0F);
```
Comment (primetoxinz) [inline src/main/java/betterwithmods/util/item/ItemExt.java]: should be "slimeball"

**Comment by BeetoGuy** [comment]
Eh, the slime ore dictionary thing is an easy enough fix.

**Pull request merged**